### PR TITLE
Set up Capstone Project - Module 2

### PR DESCRIPTION
**Pull request by angeluray**
## Hi dear good reviewer 🤩

La Rachel

## Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added CSS files that include Capstone project styles.
- Added JS files and Modules that include the logic process and structure.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Built using the Webpack.
- Added tests using Jest.

## Project Requirements Basic checks ✔

- Created both interfaces Homepage and Popup window.
- Created tests module.
- Used APIs.

### CODE REVIEW

**Comment by Mov305** [comment]
Fixed the linters please wait for it

File: README.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ Awesome Movies list, you can add your movies, like them, and reserve them.
```
Comment (Deepakdanger) [inline README.md:5]: - [ ] Kindly add some more descriptions here about the website.
- [ ] [ OPTIONAL] KIndly change the title name that looks like the website name.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 
- The design and CSS look much good :+1: 
- Nice way to display the selected movie :100: 

## Required Changes ♻️
- [ ] KIndly separate the author's details in separate lines in the readme file.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It would be nice if you have a separate file where all the API calls are written and import the necessary API function in the different files as required. It will make the code look better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @angeluray, @Mov305 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive readme
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**